### ISSUE TITLE
Improve accessibility of messages in the UI

### ISSUE THREAD

**Issue by mmbotelho** [okTurtles/group-income#504]
### Problem
The `message` component relies solely on color to convey meaning. This means, for example, that the difference between a **warning** and a **danger** message is solely conveyed by using **orange** or **red**. This can be a problem for color blind users. You can read more about this issue [here](https://uxmag.com/articles/usability-tip-dont-rely-on-color-to-convey-your-message). The UI of warnings and danger messages also look different.

From what I've been able to see in our code, we are currently using 2 types of messages:

- `.message .is-warning`
![screen shot 2018-11-21 at 13 22 20](https://user-images.githubusercontent.com/22004566/48952870-17264a00-ef44-11e8-935a-c7e10a4c58db.png)

- `.message .is-danger`
![screen shot 2018-11-23 at 17 07 44](https://user-images.githubusercontent.com/22004566/48952879-21484880-ef44-11e8-8dd7-fa04f2cf8a96.png)

### Solution
To improve the readability of the interface for color blind users, I suggest:

- Adding an icon to the message;
- Making the UI consistent, so users don't have to get accustomed to 2 types of layout for messages;
- Using black on top of the light colors, to increase contrast and help users with low vision;
- I've also designed a`.message .is-info`, in case it is needed in the future;

![screen shot 2018-11-23 at 17 40 58](https://user-images.githubusercontent.com/22004566/48953694-fad7dc80-ef46-11e8-996f-c9ffa3378186.png)

For information on spacing, colors, font-size and icons used, [visit the components page in Figma](https://www.figma.com/file/ZR0DRPhhUuHdJtf7PbGAM6tW/Group-Income?node-id=48%3A60)! Let me know if you find any other `.message` variations that need to be reviewed!

**Comment by taoeffect** [comment]
Should be closed by #523
